Added reproduction log entry

ErrenYeager opened the pull request:
System environment: RAM 16, Ubuntu 22.04
Everything worked as expected in the start-here and other sections well except for when I tried to run this command "mvn clean package appassembler:assemble" it got stuck at a certain point.

And about the documentations, everything was great and clear, however as a suggestion the section about reproduction log is a little bit confusing compared to the other parts; if there were more details about what has been asked, it would be wonderful.

Best regards,
Erfan Parifard

lintool commented:
Can you look at, for example, https://github.com/castorini/anserini/pull/2176 ?

ErrenYeager commented:
> Can you look at, for example, #2176 ?
i7, 12th gen
Python version: 3.10.6
Java Version: 17.0.7 2023-04-18
Maven Version: Apache Maven 3.6.3

lintool commented:
You've placed the reproduction entry in the wrong place.

ErrenYeager commented:
I am so sorry, I fixed it like the example.

On Sat, Sep 2, 2023 at 9:29 PM Jimmy Lin ***@***.***> wrote:

> You've placed the reproduction entry in the wrong place.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/castorini/anserini/pull/2180#issuecomment-1703900865>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY7AHS6DA46VII233KG6523XYNXZRANCNFSM6AAAAAA4IW7TVE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

lintool commented on docs/reproducibility.md:
```diff
@@ -15,3 +15,5 @@ For the most part, replicability is not applicable in the context of Anserini, b
 
 At the bottom of many pages you'll find a "Reproduction Log", which keeps track of users who have successfully reproduced the results reported on that page.
 Note that we stretch the meaning of "same team" a bit in these logs: we still consider it a successful reproduction if another member of our research group is able to obtain the same results, as long as the person was not the primary author of the code in question.
+
```
Please revert the edits to this file; i.e., remove extra lines.

ErrenYeager commented:
Dear Professor Lin

I have made updates to the project's documentation. I've reverted changes
in the mentioned file and added a new file in the "docs" directory called
"reproduction-details.md." This new file provides additional guidelines for
students on how to submit a reproduction log.

I believe these changes will improve the project's documentation. Please
review the new file, and if you find it suitable, consider incorporating it
into the project.

Thank you for your time and consideration.

Best regards,
Erfan Parifard

On Sun, Sep 3, 2023 at 12:45 AM Jimmy Lin ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In docs/reproducibility.md
> <https://github.com/castorini/anserini/pull/2180#discussion_r1313993297>:
>
> > @@ -15,3 +15,5 @@ For the most part, replicability is not applicable in the context of Anserini, b
>
>  At the bottom of many pages you'll find a "Reproduction Log", which keeps track of users who have successfully reproduced the results reported on that page.
>  Note that we stretch the meaning of "same team" a bit in these logs: we still consider it a successful reproduction if another member of our research group is able to obtain the same results, as long as the person was not the primary author of the code in question.
> +
>
> Please revert the edits to this file; i.e., remove extra lines.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/castorini/anserini/pull/2180#pullrequestreview-1608380764>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY7AHS2N4MW2PTM6CTKZKXDXYOOW7ANCNFSM6AAAAAA4IW7TVE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
